shuttle/templates/schedules/request_shuttle.html

```diff
@@ -11,31 +11,61 @@ <h4>Select a location and time below to request the shuttle:</h4>
     <div class="center-page">
         <div class="align-side-by-side">
             <div class="dropdown">
-                <button class="btn btn-primary btn-lg dropdown-toggle" type="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Location</button>
+                <button class="btn btn-primary btn-lg dropdown-toggle location-menu" type="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Location</button>
                 <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
-                    <a class="dropdown-item" href="#">*Insert location here*</a>
-                    <a class="dropdown-item" href="#">*Insert more locations*</a>
-                </div>
-            </div>
-        </div>
-        <div class="align-side-by-side">
-            <div class="dropdown">
-                <button class="btn btn-primary btn-lg dropdown-toggle" type="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Time</button>
-                <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
-                   <a class="dropdown-item" href="#">*Insert time here*</a>
-                    <a class="dropdown-item" href="#">*Insert more times here*</a>
+                    {% for location in locations %}
+                        <a class="dropdown-item" href="#">{{ location }}</a>
+                    {% endfor %}
                 </div>
             </div>
         </div>
         <div class="align-side-by-side">
             <div class="submit">
-                <input class="btn btn-warning btn-lg" type="submit" value="Request Shuttle">
+                <input class="btn btn-warning btn-lg submit-btn" type="submit" value="Request Shuttle">
             </div>
         </div>
     </div>
 
     <div class="center-page">
-       <h4>Students awaiting shuttle: <span class="badge badge-light">*Insert number here*</span></h4>
+       <h4>Current Shuttle Waitlist: <span class="badge badge-pill badge-warning"></span></h4>
     </div>
+    <script>
+    function waitlist() {
+        // Updates waitlist
+        $.ajax({
+              url: "{{ url_for('SchedulesView:check_waitlist') }}",
+        })
+            .done(function(data) {
+                $(".badge").text(data);
+            })
+            .fail(function () {
+                alert("The waitlist is not responding. Please call the ITS Help Desk at 651-638-6500 for support");
```

Comment:
Don't use pop up alerts, they are bad.
https://ux.stackexchange.com/a/72642

Instead use in page alerts. The project is already setup to use them.

In Python when you know something fails call self.sc.set_alert() with the correct parameters (method lives in shuttle controller) and then in javascript when we fail, reload the page and the error will be displayed.

```@route('/create', methods=['POST'])
    def create_new_time_slot(self):
        self.wcc.check_roles_and_route(['Administrator'])
        form = request.form

        start_time = form.get('start-time')
        start_time = datetime.strftime(datetime.strptime(start_time, '%H:%M'), '%H:%M:%S')

        end_time = form.get('end-time')
        end_time = datetime.strftime(datetime.strptime(end_time, '%H:%M'), '%H:%M:%S')

        is_active = form.get('active')

        created = self.sc.create_time_slot(start_time, end_time, is_active)

        if created:
            self.wcc.set_alert('success', 'Schedule Created Successfully!')
        else:
            self.wcc.set_alert('danger', 'Schedule already exists!')
        return redirect(url_for('SchedulesView:create_time_slot'))
```

This is an example of the Python side from https://github.com/betheluniversity/writing-center-python. Feel free to reference it or ask questions.